src/components/navigation/NavBar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { Navbar, Container, Nav } from 'react-bootstrap';
+import logo from '../images/assets/planet.png';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const NavBar = () => (
+  <>
+    <Navbar className="border-bottom">
+      <Container className="nav-wrapper">
+        <Navbar.Brand href="/">
+          <img src={logo} alt="Logo" className="logo" />
+          <h1>Space Travelers Hub</h1>
+        </Navbar.Brand>
+        <Nav className="nav-links p-3">
```

Comment:
- [ ] Great job so far. design looks great. please ensure that the active page is  indicated with an underline
![Screenshot (484)_LI](https://user-images.githubusercontent.com/65298048/193288262-396843e8-9f09-41fe-a2d5-6b1924f883f3.jpg)